irida_staramr_results/config/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+DEFAULT_CONFIG_PATH = os.path.dirname(os.path.abspath(__file__))
```

Comment:
I'm not sure if there are better ways to do this, which assigns ` irida_staramr_results/config` as the default config path.

Also, I noticed the missing empty line at the end of the file. I'll fix it asap.